shared/constants/network.ts

```diff
@@ -1077,6 +1077,24 @@ export const FEATURED_RPCS: AddNetworkFields[] = [
   },
 ];
 
+export const allowedInfuraHosts = [
+  'mainnet.infura.io',
+  'linea-mainnet.infura.io',
+  'polygon-mainnet.infura.io',
+  'base-mainnet.infura.io',
+  'blast-mainnet.infura.io',
+  'optimism-mainnet.infura.io',
+  'arbitrum-mainnet.infura.io',
+  'palm-mainnet.infura.io',
+  'avalanche-mainnet.infura.io',
+  'celo-mainnet.infura.io',
+  'zksync-mainnet.infura.io',
+  'bsc-mainnet.infura.io',
+  'mantle-mainnet.infura.io',
+  'opbnb-mainnet.infura.io',
+  'scroll-mainnet.infura.io',
```

Comment:
testnet are not included here

Reply:
where did you get this list?

Reply:
i got it from infura doc [here](https://docs.metamask.io/services/get-started/endpoints/)

Reply:
testnet are excluded now